millw.bat

```diff
@@ -37,6 +37,9 @@ if [!MILL_VERSION!]==[] (
   if exist .mill-version (
       set /p MILL_VERSION=<.mill-version
   )
+  else if exist .config\mill-version (

```

Comment:
been a long time since I wrote batch, check if this is correct

Reply:
I think we need extra parenthesis in the else section, but have currently no system to test it.